static/js/components/Candidate.jsx

```diff
@@ -92,8 +92,12 @@ const Candidate = ({ route }) => {
                 {candidate.gal_lat.toFixed(6)}
                 )
                 <br />
-                <b>Redshift: &nbsp;</b>
-                {candidate.redshift.toFixed(4)}
+                {candidate.redshift != null && (
```

Comment:
```suggestion
                {candidate.redshift && (
```

Reply:
No that doesn't work. If z=0 then this doesn't pass